src/new/wallet.js

```diff
@@ -768,7 +768,7 @@ class HathorWallet extends EventEmitter {
         version: tx.version,
         ncId: tx.nc_id,
         ncMethod: tx.nc_method,
-        ncCaller: tx.nc_pubkey && getAddressFromPubkey(tx.nc_pubkey),
+        ncCaller: tx.nc_pubkey && getAddressFromPubkey(tx.nc_pubkey, this.getNetworkObject()),
```

Comment:
We should have an integration test for this, since it was a bug before.

Reply:
Do we have a unit test at least?
If so, why did it not warn us of this?

Reply:
We can use the integration tests that we already have for nano contracts at https://github.com/HathorNetwork/hathor-wallet-lib/blob/master/__tests__/integration/nanocontracts/bet.test.js to call this method at least once to validate it. 

What do you think?

Reply:
Done.